snapcraft/cli/_errors.py

```diff
@@ -42,10 +54,15 @@ def exception_handler(exception_type, exception, exception_traceback, *,
     exit_code = 1
     is_snapcraft_error = issubclass(exception_type, errors.SnapcraftError)
 
-    if debug or not is_snapcraft_error:
+    if debug:
         traceback.print_exception(
             exception_type, exception, exception_traceback)
 
+    if not is_snapcraft_error:
+        echo.error('Sorry, Snapcraft had an internal error.')
+        if RavenClient is not None and _is_environment_ok():
+            _submit_trace(exception)
```

Comment:
When raven is not available, the message about sorry feels very empty. Shouldn't we give a link to report the issue, at least? Or point people to the forum to talk about it.

Reply:
This will only happen with the deb. This is according to the spec we already designed.